test_result.txt

```diff
@@ -0,0 +1,65 @@
+============================= test session starts ==============================
```

Comment:
Aha, gotcha. Good idea, but I'd suggest doing that by putting the log in a gist and linking to it. Anything that's committed to the repository ends up in everyone's copy of the git history for ever, unless we fiddle around with rebasing and squashing commits, so we tend not to commit things that we intend to delete again.